### DISCUSSION TITLE
3.4.0

### DISCUSSION THREAD

**Discussion by Saranya-jena** [litmuschaos/litmus#4441] [Releases]
## What's Changed
* chore(deps): bump github.com/google/uuid from 1.3.0 to 1.6.0 in /chaoscenter/authentication by @dependabot in https://github.com/litmuschaos/litmus/pull/4399
* chore(deps): bump google.golang.org/grpc from 1.32.0 to 1.61.0 in /chaoscenter/authentication by @dependabot in https://github.com/litmuschaos/litmus/pull/4408
* chore(deps): bump golang.org/x/crypto from 0.5.0 to 0.17.0 in /chaoscenter/graphql/server by @dependabot in https://github.com/litmuschaos/litmus/pull/4403
* chore(deps): Bump golang.org/x/crypto from 0.0.0-20220315160706-3147a52a75dd to 0.17.0 in /litmus-portal/cluster-agents/event-tracker by @dependabot in https://github.com/litmuschaos/litmus/pull/4346
* chore(deps): Bump golang.org/x/crypto from 0.13.0 to 0.18.0 in /chaoscenter/authentication by @dependabot in https://github.com/litmuschaos/litmus/pull/4373
* chore(deps): Bump github.com/sirupsen/logrus from 1.8.1 to 1.9.3 in /chaoscenter/event-tracker by @dependabot in https://github.com/litmuschaos/litmus/pull/4275
* chore(deps): Bump google.golang.org/grpc from 1.44.0 to 1.56.3 in /chaoscenter/subscriber by @dependabot in https://github.com/litmuschaos/litmus/pull/4269
* chore(deps): Bump github.com/sirupsen/logrus from 1.8.1 to 1.9.3 in /chaoscenter/subscriber by @dependabot in https://github.com/litmuschaos/litmus/pull/4284
* chore(deps): Bump github.com/sirupsen/logrus from 1.9.2 to 1.9.3 in /chaoscenter/authentication by @dependabot in https://github.com/litmuschaos/litmus/pull/4276
* chore(deps): Bump go.mongodb.org/mongo-driver from 1.5.3 to 1.13.1 in /chaoscenter/authentication by @dependabot in https://github.com/litmuschaos/litmus/pull/4413
* chore(deps): Bump golang.org/x/oauth2 from 0.14.0 to 0.16.0 in /chaoscenter/authentication by @dependabot in https://github.com/litmuschaos/litmus/pull/4412
* feat: Automatically generate probes when creating a chaos experiment using a YAML file. by @namkyu1999 in https://github.com/litmuschaos/litmus/pull/4366
* Fixes #4354: fixed gql schema for environment filter input  by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4415
* refactor: Update subscriber to use Kubernetes patch for ConfigMaps an… by @RipulHandoo in https://github.com/litmuschaos/litmus/pull/4287
* Adds projectID in listInfras gql response by @Nageshbansal in https://github.com/litmuschaos/litmus/pull/4393
* chore: Remove conditions for zero case by @namkyu1999 in https://github.com/litmuschaos/litmus/pull/4394
* chore(deps): Bump golang.org/x/crypto from 0.0.0-20220315160706-3147a52a75dd to 0.17.0 in /litmus-portal/upgrade-agents/control-plane by @dependabot in https://github.com/litmuschaos/litmus/pull/4345
* chore: [CHAOS-3808]: Resilience probe modal toggle reseting to default bugfix by @S-ayanide in https://github.com/litmuschaos/litmus/pull/4431
* Added Fuzz test suite for Get Environment API by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4437
* Fixes getServerVersion GQL by @Nageshbansal in https://github.com/litmuschaos/litmus/pull/4424
* [Proposal] : Litmusctl Probe CRUD Commands by @Nageshbansal in https://github.com/litmuschaos/litmus/pull/4387
* Fixed issue infra config map and secret patch by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4440
* updated the environment_id index to be a partial index by @SarthakJain26 in https://github.com/litmuschaos/litmus/pull/4436
* Added 3.4.0 installation files by @Saranya-jena in https://github.com/litmuschaos/litmus/pull/4439

**Full Changelog**: https://github.com/litmuschaos/litmus/compare/3.3.0...3.4.0

<hr /><em>This discussion was created from the release <a href='https://github.com/litmuschaos/litmus/releases/tag/3.4.0'>3.4.0</a>.</em>

**Comment by amoughit** [comment]
Hi, is it possible to upgrade from 3.3.0 to 3.4.0 ?
